packages/server/graphql/types/TeamPromptMeeting.ts

```diff
@@ -0,0 +1,44 @@
+import {GraphQLList, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import toTeamMemberId from 'parabol-client/utils/relay/toTeamMemberId'
+import {getUserId} from '../../utils/authorization'
+import {GQLContext} from '../graphql'
+import TeamPromptMeetingMember from './TeamPromptMeetingMember'
+import ActionMeetingSettings from './ActionMeetingSettings'
+import NewMeeting, {newMeetingFields} from './NewMeeting'
+import Task from './Task'
+
+const TeamPromptMeeting = new GraphQLObjectType<any, GQLContext>({
+  name: 'TeamPromptMeeting',
+  interfaces: () => [NewMeeting],
+  description: 'A team prompt meeting',
+  fields: () => ({
+    ...newMeetingFields(),
+    settings: {
+      type: new GraphQLNonNull(ActionMeetingSettings),
+      description: 'The settings that govern the action meeting',
```

Comment:
+1 s/action/team prompt/

Reply:
nice, so I actually fixed that copy/paste issue [here](https://github.com/ParabolInc/parabol/pull/6082/files#diff-8ef25ccca4f474290656cc4001f51fd6fa90a6141b33732de3db2e1e99ac26d7R11) and then, did the same myself 🤦🏻‍♂️